modules/context/repo.go

```diff
@@ -17,6 +17,7 @@ import (
 	"code.gitea.io/gitea/models/db"
 	git_model "code.gitea.io/gitea/models/git"
 	issues_model "code.gitea.io/gitea/models/issues"
+	packages_model "code.gitea.io/gitea/models/packages"
 	access_model "code.gitea.io/gitea/models/perm/access"
 	repo_model "code.gitea.io/gitea/models/repo"
```

Comment:
This file needs a **serious** refactoring.
A `modules/context` file, which is one of the most essential files in the hierarchy, should not depend on `models` and `services`…
But that's not a problem for this PR.